commands/signin.js

```diff
@@ -41,11 +41,11 @@ function exchangeAuthCodeForAccessToken (authorizationCode) {
     url: exchangeUri,
     headers: { 'content-type': 'application/json' },
     body: JSON.stringify({
-      "grant_type": "authorization_code",
-      "client_id": clientId,
-      "code_verifier": verifier,
-      "code": authorizationCode,
-      "redirect_uri": redirectUri
+      'grant_type': 'authorization_code',
```

Comment:
sure, I'm aware... the way this pattern plays out is that some JSON gets pasted in there, then variables threaded in, then someone notices that it doesn't lint with the double-quotes...

I'll change it to match your suggestion here